exporter/splunkhecexporter/buffer.go

```diff
@@ -71,6 +74,9 @@ func (b *bufferState) Close() error {
 
 // accept returns true if data is accepted by the buffer
 func (b *bufferState) accept(data []byte) (bool, error) {
+	if len(data)+b.rawLength > int(b.maxEventLength) {
```

Comment:
this is incorrect. I should have caught this.

Proper would have been:
```
if len(data) > int(b.maxEventLength) {
```

Reply:
okay, just to be clear, we're moving from per batch to per event. is that correct?

Reply:
correct